pages/linux/btrfs-balance.md

```diff
@@ -0,0 +1,32 @@
+# btrfs balance
+
+> Balance block groups on a btrfs filesystem.
+> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-balance>.
+
+- Show the status of a running or paused balance:
+
+`sudo btrfs balance status {{path/to/btrfs_filesystem}}`
+
+- Balance all block groups (slow; rewrites all blocks in filesystem):
+
+`sudo btrfs balance start {{path/to/btrfs_filesystem}}`
+
+- Balance data block groups less than 15% utilized (and run in background):
```

Comment:
Perhaps this will read better?
```suggestion
- Balance data block groups which are less than 15% utilized, running the operation in the background:
```